parameters/production-bounty-dao.json

```diff
@@ -0,0 +1,100 @@
+{
+  "orgName": "BountyDAO",
+  "tokenName": "BountyDAO Token",
+  "tokenSymbol": "BDT",
+  "ContributionReward": [{}],
+  "SchemeRegistrar": [{}],
+  "StandAloneContracts": [
+    {
+      "name": "PolkaCurve",
+      "fromArc": true,
+      "arcVersion": "0.0.1-rc.39"
+    },
+    {
+      "name": "ContributionRewardExt",
+      "fromArc": true,
+      "arcVersion": "0.0.1-rc.39"
+    },
+    {
+      "name": "Competition",
+      "fromArc": true,
+      "params": [{ "StandAloneContract": 1 }],
+      "arcVersion": "0.0.1-rc.39"
+    }
+  ],
+  "VotingMachinesParams": [
+    {
+      "boostedVotePeriodLimit": 43200,
+      "daoBountyConst": 10,
+      "minimumDaoBounty": 150,
+      "queuedVotePeriodLimit": 604800,
+      "queuedVoteRequiredPercentage": 50,
+      "preBoostedVotePeriodLimit": 43200,
+      "proposingRepReward": 250,
+      "quietEndingPeriod": 3600,
+      "thresholdConst": 1200,
+      "voteOnBehalf": "0x0000000000000000000000000000000000000000",
+      "votersReputationLossRatio": 4,
+      "activationTime": 1580212800
+    },
+    {
+      "boostedVotePeriodLimit": 28800,
+      "daoBountyConst": 10,
+      "minimumDaoBounty": 150,
+      "queuedVotePeriodLimit": 604800,
+      "queuedVoteRequiredPercentage": 50,
+      "preBoostedVotePeriodLimit": 28800,
+      "proposingRepReward": 250,
+      "quietEndingPeriod": 3600,
+      "thresholdConst": 1200,
+      "voteOnBehalf": "0x0000000000000000000000000000000000000000",
+      "votersReputationLossRatio": 4,
+      "activationTime": 0
+    }
+  ],
+  "schemes": {
+    "ContributionReward": true,
+    "SchemeRegistrar": true,
+    "GlobalConstraintRegistrar": false,
+    "UpgradeScheme": false
+  },
+  "CustomSchemes": [
+    {
+      "name": "ReputationFromToken",
+      "isUniversal": false,
+      "params": [
+        "0x30412F4c06343d65C06B5a0Bba6821cf37E605fa",
+        { "StandAloneContract": 0 },
+        "0x0000000000000000000000000000000000000001"
```

Comment:
please make it 0x0000000000000000000000000000000000000000

Reply:
done